### ISSUE TITLE
Action.Execute field "associatedInputs" issue with case sensitivity

### ISSUE THREAD

**Issue by Wajeed-msft** [MicrosoftDocs/msteams-docs#6175]
Copied bug from: https://github.com/microsoft/AdaptiveCards/issues/7213 

**Details from original post:**
### Target Application

Microsoft Teams

### Application Operating System

Other

### Schema Version

1.5

### Problem Description

associatedInputs in the docs https://adaptivecards.io/explorer/Action.Execute.html has such description listed:
`Allowed values:
"Auto": Inputs on the current card and any parent cards will be validated and submitted for this Action.
"None": None of the inputs will be validated or submitted for this Action.`

And in [designer playground](https://adaptivecards.io/designer) `"associatedInputs": "None"` works fine. But when I was implementing it via Json (I am using Scala, not C#), sending Json card structure to the Microsoft Teams app, the property wasn't working until I made the field `"associatedInputs": "none"` without capital letter. This feels like either an issue when it comes to the way Json is processed or issues with documentation/designer tools, that behaviour isn't consistent.

### Expected Outcome

associatedInputs field works with "None" string value assigned or documents/designer appropriately doesn't work with "None" and don't say this is the needed value.

### Actual Outcome

associatedInputs field doesn't work with "None" string value assigned, while documents/designer work fine with "None" value assigned

### Card JSON

```JSON
{
   "type":"application/vnd.microsoft.card.adaptive",
   "value":{
      "type":"AdaptiveCard",
      "body":[
         {
            "text":"Choose search object type",
            "type":"TextBlock"
         },
         {
            "id":"objectType",
            "choices":[
               {
                  "title":"Opportunity",
                  "value":"Opportunity"
               },
               {
                  "title":"Account",
                  "value":"Account"
               },
               {
                  "title":"Case",
                  "value":"Case"
               }
            ],
            "style":"filtered",
            "isRequired":true,
            "type":"Input.ChoiceSet"
         }
      ],
      "version":"1.5",
      "refresh":{
         "action":{
            "isEnabled":true,
            "associatedInputs":"auto",
            "type":"Action.Execute"
         },
         "userIds":[
            
         ]
      },
      "actions":[
         {
            "data":{
               "entityId":"someId",
               "userId":"someId",
               "tenant":"someId"
            },
            "title":"Confirm",
            "verb":"typeChosen",
            "isEnabled":true,
            "associatedInputs":"auto",
            "type":"Action.Execute"
         },
         {
            "data":{
               "entityId":"someId",
               "userId":"someId",
               "tenant":"someId"
            },
            "title":"Back",
            "verb":"backToYes",
            "isEnabled":true,
            "associatedInputs":"None",
            "type":"Action.Execute"
         }
      ]
   }
}
```

### Repro Steps

_No response_---
#### Document Details

⚠ *Do not edit this section.It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3fb4394d-25b1-8752-3996-fe3a2934732d
* Version Independent ID: 6d391381-89d0-117c-27cc-00043bd26a36
* Content: [Cards reference - Teams](https://docs.microsoft.com/en-us/microsoftteams/platform/task-modules-and-cards/cards/cards-reference)
* Content Source: [msteams-platform/task-modules-and-cards/cards/cards-reference.md](https://github.com/MicrosoftDocs/msteams-docs/blob/master/msteams-platform/task-modules-and-cards/cards/cards-reference.md)
* Product: **msteams**
* GitHub Login: @surbhigupta12
* Microsoft Alias: **surbhigupta12**

**Comment by v-chetsh** [comment]
@limar63 - We will investigate on it and get back to you.

**Comment by Meghana-MSFT** [comment]
@limar63 - Are you facing this issue only with Scala?

**Comment by limar63** [comment]
I don't think that this has anything to do with Scala because I am working with pure JSON there. There aren't any libraries (at least haven't seen any) for adaptive cards on Scala and using ZIO JSON encoder to turn case classes into JSON files and issue is with reading JSON payload by microsoft teams itself from what I understand.

**Comment by Meghana-MSFT** [comment]
@limar63 - Could you please help by sharing logs and also a recording of the repro.

**Comment by Meghana-MSFT** [comment]
@limar63 - Could you please help by sharing logs and also a recording of the repro.

**Comment by limar63** [comment]
Yes, sry for not seeing the first message. I will be able to dig into it on the Monday, right now have a very busy working week with not enough time and not working with ms teams so would need to invest some time to get back the result.

**Comment by Meghana-MSFT** [comment]
@limar63 - Did you get a chance to repro the issue again?

**Comment by ChetanSharma-msft** [comment]
Hello @limar63 - Hope you are doing well !!
Did you get some time to check or repro the issue ?

Please share your inputs, so that we can further investigate on this issue.
